hermes-console-vue/src/composables/use-stats/useStats.ts

```diff
@@ -0,0 +1,92 @@
+import { fetchStats } from '@/api/hermes-client';
+import { ref } from 'vue';
+import type { Ref } from 'vue';
+
+export interface UseStats {
+  topicCount: Ref<number | undefined>;
+  ackAllTopicCount: Ref<number | undefined>;
+  ackAllTopicShare: Ref<number | undefined>;
+  trackingEnabledTopicCount: Ref<number | undefined>;
+  trackingEnabledTopicShare: Ref<number | undefined>;
+  avroTopicCount: Ref<number | undefined>;
+  avroTopicShare: Ref<number | undefined>;
+  subscriptionCount: Ref<number | undefined>;
+  trackingEnabledSubscriptionCount: Ref<number | undefined>;
+  trackingEnabledSubscriptionShare: Ref<number | undefined>;
+  avroSubscriptionCount: Ref<number | undefined>;
+  avroSubscriptionShare: Ref<number | undefined>;
+  loading: Ref<boolean>;
+  error: Ref<boolean>;
+}
+
+export function useStats(): UseStats {
+  const loading = ref(false);
+  const topicCount = ref<number>();
```

Comment:
You can define an interface that will contain all those statistics. This would reduce all those refs to just one ref named statistics, that will be returned from this composable

Reply:
fixed